File: Day23/Solution.cpp
void levelOrder(Node * root){
      //Write your code here
        if (root == NULL)
            return;
        queue<Node*> q;
        q.push(root);
        while(!q.empty()){
            Node* current = q.front();
            cout<< current->data << " ";
            if(current->left != NULL)
                q.push(current->left);

            if(current->right !=NULL)
                q.push(current->right);

            q.pop();
        }
	}

File: Day23/Solution.py
    def levelOrder(self,root):
        output = ""
        queue = [root]
        while queue:
            current = queue.pop(0)
            output += str(current.data) + " "
            if current.left:
                queue.append(current.left)
            if current.right:
                queue.append(current.right)
        print(output[:-1])